skops/card/_model_card.py

```diff
@@ -410,9 +410,12 @@ def _generate_card(self) -> ModelCard:
                         'clf.predict(pd.DataFrame.from_dict(config["sklearn"]["example_input"]))'
                     )
         if self.model_diagram is True:
-            model_plot: str | None = re.sub(
+            model_plot_div = re.sub(
                 r"\n\s+", "", str(estimator_html_repr(self.model))
             )
+            assert model_plot_div.count("sk-top-container") == 1
+            model_plot_div = model_plot_div.replace("sk-top-container", 'sk-top-container" style="overflow: auto;')
```

Comment:
Neat 🙂 
Maybe a test that asserts this overflow style is in model card would be nice! (you can look from some model card tests for inspiration) If you need any help in writing tests please let us know hug🤗